databases/week1/hw-week1.sql

```diff
@@ -0,0 +1,53 @@
+USE hyf;
+-- 1Find out how many tasks are in the task table
+SELECT COUNT(id)
+ FROM task;
+ 
+-- 2  find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(id) 
+FROM task
+WHERE due_date IS null;
+
+-- 3  Find all the tasks that are marked as done
+SELECT title 
+FROM task 
+WHERE status_id = 3;
+
+-- 4  Find all the tasks that are not marked as done
+SELECT title 
+FROM task 
+WHERE status_id != 3;
+
+-- 5 Get all the tasks, sorted with the most recently created first
```

Comment:
Will this sort with the most recent created displayed first? If you have two rows with created at dates like this:

    1. 2020-09-21
    2. 2020-09-22

then row 2 should be displayed first. https://www.w3schools.com/sql/sql_orderby.asp

Try to have consistent code foramtting and write `ORDER BY` in uppercase.